data-and-programming.Rmd

```diff
@@ -120,53 +122,73 @@ length(y)
 
 To subset a vector, we use square brackets, `[]`. 
 
-```{r}
-x
+```{r results = 'hold'}
+x = c(1, 3, 5, 7, 8, 9)
```

Comment:
I'm not sure defining `x` in multiple chunks is a good idea.

How about create an assignment chunk and then embed it in each chunk?

````
```{r init-code}

```


```{r my-chunk}
<<init-code>>

# Work here
# Note: This approach does not work when individually running code chunks.
```
````

Reply:
I'd generally like the document to function as expected if the reader were to download and run locally.